includes/class-wcpdf-admin.php

```diff
@@ -382,7 +382,7 @@ public function data_input_box_content( $post ) {
 			$this->output_number_date_edit_fields( $invoice, $data );
 
 		} else {
-			printf( '<div class="wcpdf-data-fields"><p>&#9888; %s</p></div>', __( 'Invoice not generated for this order.', 'woocommerce-pdf-invoices-packing-slips' ) );
+			printf( '<div class="wcpdf-data-fields"><p>&#9888; %s</p></div>', __( 'Some error ocurred trying to access the Invoice document.', 'woocommerce-pdf-invoices-packing-slips' ) );
```

Comment:
That sounds worse than it is. Wouldn't this already appear if the invoice is simply disabled (0.00 order, disabled status, etc)? Do we need an error at all (previous versions didn't have it)?